server/src/main/java/com/phoenixoft/teambalanceapp/controller/AuthenticationController.java

```diff
@@ -33,8 +32,10 @@ public AuthenticationResponse createAuthenticationToken(@RequestBody Authenticat
     }
 
     @PostMapping(path = "/register")
-    public UserResponseDto registerNewUser(@RequestBody @Valid UserRegistrationRequestDto dto) {
-        return DtoConverter.convertUser(userService.registerNewUser(dto));
+    public AuthenticationResponse registerNewUser(@RequestBody @Valid UserRegistrationRequestDto dto) {
+        userService.registerNewUser(dto);
+        AuthenticationRequest authenticationRequest = new AuthenticationRequest(dto.getEmail(), dto.getPassword());
+        return authenticationService.authenticateUser(authenticationRequest);
```

Comment:
We can just call `jwtTokenUtil.generateToken(user)` to generate JWT for newly created user. What do you think?

Reply:
Yes, you are right